lx_anonymizer/llm_phi4.py

```
@@ -1,43 +1,47 @@
+
+from .custom_logger import get_logger
 from .model_service import model_service
+from .ocr import cleanup_gpu
 
 logger = get_logger(__name__)
 
+
 def phi4_ocr_correction(text):
     """Verbessert OCR-erkannten Text mit Phi-4"""
     # Lade Modelle vom zentralen Service
     model, tokenizer, pipe = model_service.load_phi4_model()
```

Ruff (0.14.1)

Unpacked variable `pipe` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)